### ISSUE TITLE
Segmentation fault on dbms_sql.column_value with null value

### ISSUE THREAD

**Issue by chmyaf** [orafce/orafce#257]
### Environment
orafce: VERSION_4_9_2 and master
postgresql: REL_16_2

### Steps to reproduce

```
create extension orafce;
create table test(id text);
insert into test(id) values ('1'), (null);
```

```
do
$$
    declare
        cursor int;
        id text;
        row_counter int := 0;
    begin
        cursor := dbms_sql.open_cursor();
        call dbms_sql.parse(cursor, 'select id from test');
        call dbms_sql.define_column(cursor, 1, 'id');
        perform dbms_sql.execute(cursor);
        while dbms_sql.fetch_rows(cursor) > 0 loop
                row_counter = row_counter + 1;
                raise notice 'process row #%', row_counter;
                call dbms_sql.column_value(cursor, 1, id);
                raise notice 'row id: `%`', id;
            end loop;
        call dbms_sql.close_cursor(cursor);
    end;
$$
```

### Backtrace
```
[New LWP 115542]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `postgres: postgres postgres 127.0.0.1(38602) DO                  '.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  datumTransfer (value=0, typByVal=false, typLen=-1) at ../../../../src/include/postgres.h:314
314		return (Pointer) X;
#0  datumTransfer (value=0, typByVal=false, typLen=-1) at ../../../../src/include/postgres.h:314
No locals.
#1  0x0000561cdc954590 in SPI_datumTransfer (value=<optimized out>, typByVal=<optimized out>, typLen=<optimized out>) at spi.c:1369
        oldcxt = 0x561cded3b330
        result = <optimized out>
        __func__ = "SPI_datumTransfer"
#2  0x00007f826e42e99c in dbms_sql_column_value (fcinfo=0x7fff419285d0) at dbms_sql.c:1755
        c = 0x7f826e440c80 <cursors>
        value = 94681997645912
        result = <optimized out>
        pos = 1
        isnull = true
        targetTypeId = <optimized out>
        resultTypeId = 2249
        resulttupdesc = 0x561cded3b570
        resulttuple = <optimized out>
        oldcxt = 0x561cded3d340
        __func__ = "dbms_sql_column_value"
#3  0x0000561cdc8c8121 in ExecuteCallStmt (stmt=stmt@entry=0x561cdecfc658, params=params@entry=0x561cded09e78, atomic=atomic@entry=false, dest=dest@entry=0x561cdced5440 <spi_printtupDR>) at functioncmds.c:2286
        fcinfodata = {fcinfo = {flinfo = 0x7fff419285a0, context = 0x561cdecfcbb0, resultinfo = 0x0, fncollation = 100, isnull = false, nargs = 3, args = 0x7fff419285f0}, fcinfo_data = "\240\205\222A\377\177\000\000\260\313\317\336\034V\000\000\000\000\000\000\000\000\000\000d\000\000\000\000V\003\000\000\000\000\000\000\000\000\000\000\304\317\336\000\000\000\000\001\000\000\000\000\000\000\000\000\241\216\217\336\331\032\355\350\236\320\336\034V\000\000\000\000\000\000\000\000\000\000\330W\335\334\034V\000\000\200\215\222A\377\177\000\000\274\345Rn\202\177\000\000\071\026\303\334\063\071\060\071\260\214\222A\377\177\000\000\000\241\216\217\336\331\032\355\374\374Rn\202\177\000\000\000\000\000\000\000\000\000\000\330W\335\334\034V\000\000@\215\222A\377\177\000\000x\215\305\334\034V\000\000v\215\305\334\034V\000\000\360\214\222A\377\177\000\000o\037\303\334\034V", '\000' <repeats 18 times>...}
        fcinfo = 0x7fff419285d0
        lc = <optimized out>
        fexpr = 0x561cdecfc910
        nargs = <optimized out>
        i = <optimized out>
        aclresult = <optimized out>
        flinfo = {fn_addr = 0x7f826e42e8e0 <dbms_sql_column_value>, fn_oid = 50655, fn_nargs = 3, fn_strict = false, fn_retset = false, fn_stats = 1 '\001', fn_extra = 0x0, fn_mcxt = 0x561cdecfbd50, fn_expr = 0x561cdecfc910}
        callcontext = <optimized out>
        estate = 0x561cded61650
        econtext = 0x561cded61858
        tp = <optimized out>
        fcusage = {fs = 0x0, save_f_total_time = {ticks = 62}, save_total = {ticks = 2}, start = {ticks = 140734293511344}}
        retval = <optimized out>
        __func__ = "ExecuteCallStmt"
#4  0x0000561cdcab64b2 in standard_ProcessUtility (pstmt=0x561cdecfc550, queryString=0x561cded5f5f0 "call dbms_sql.column_value(cursor, 1, id)", readOnlyTree=<optimized out>, context=PROCESS_UTILITY_QUERY_NONATOMIC, params=0x561cded09e78, queryEnv=0x0, dest=0x561cdced5440 <spi_printtupDR>, qc=0x7fff41928d90) at utility.c:858
        parsetree = 0x561cdecfc658
        isTopLevel = <optimized out>
        isAtomicContext = false
        pstate = 0x561cdecfcaa8
        readonly_flags = <optimized out>
        __func__ = "standard_ProcessUtility"
#5  0x0000561cdc9549f0 in _SPI_execute_plan (plan=<optimized out>, options=0x7fff41928e60, snapshot=<optimized out>, crosscheck_snapshot=<optimized out>, fire_triggers=<optimized out>) at spi.c:2697
        context = PROCESS_UTILITY_QUERY_NONATOMIC
        qc = {commandTag = CMDTAG_UNKNOWN, nprocessed = 0}
        stmt = 0x561cded6a760
        canSetTag = true
        dest = 0x561cdced5440 <spi_printtupDR>
        lc2__state = {l = <optimized out>, i = 0}
        plansource = <optimized out>
        stmt_list = 0x561cded6a718
        lc2 = <optimized out>
        lc1__state = <optimized out>
        my_res = 0
        my_processed = 0
        my_tuptable = 0x0
        res = <optimized out>
        pushed_active_snap = <optimized out>
        plan_owner = <optimized out>
        spicallbackarg = {query = 0x561cded5f5f0 "call dbms_sql.column_value(cursor, 1, id)", mode = RAW_PARSE_DEFAULT}
        spierrcontext = {previous = 0x7fff419292a0, callback = 0x561cdc952bf0 <_SPI_error_callback>, arg = 0x7fff41928d80}
        cplan = <optimized out>
        lc1 = <optimized out>
        __func__ = "_SPI_execute_plan"
        fail = <optimized out>
#6  0x0000561cdc95546f in SPI_execute_plan_extended (plan=0x561cded60a20, options=0x7fff41928e60) at spi.c:721
        res = 0
#7  0x00007f826e51bfa2 in exec_stmt_call (estate=0x7fff419292c0, stmt=0x561cded025c0) at pl_exec.c:2229
        expr = 0x561cded026b8
        before_lxid = 9
        after_lxid = <optimized out>
        paramLI = <optimized out>
        options = {params = 0x561cded09e78, read_only = false, allow_nonatomic = true, must_return_tuples = false, tcount = 0, dest = 0x0, owner = 0x561cdec5d5f0}
        rc = <optimized out>
        __func__ = "exec_stmt_call"
#8  0x00007f826e51dc23 in exec_stmts (estate=0x7fff419292c0, stmts=0x561cded023d0) at pl_exec.c:2027
        stmt = 0x561cded025c0
        rc = <optimized out>
        s__state = {l = <optimized out>, i = 2}
        save_estmt = 0x561cded028f8
        s = <optimized out>
        __func__ = "exec_stmts"
#9  0x00007f826e51eb4c in exec_stmt_while (stmt=<optimized out>, estate=<optimized out>) at pl_exec.c:2666
        value = true
        isnull = false
        rc = 0
        rc = <optimized out>
        value = <optimized out>
        isnull = <optimized out>
#10 exec_stmts (estate=0x7fff419292c0, stmts=0x561cded01688) at pl_exec.c:2047
        stmt = 0x561cded028f8
        rc = <optimized out>
        s__state = {l = <optimized out>, i = 7}
        save_estmt = 0x561cded02b20
        s = <optimized out>
        __func__ = "exec_stmts"
#11 0x00007f826e52068b in exec_stmt_block (estate=0x7fff419292c0, block=0x561cded02b20) at pl_exec.c:1942
        rc = -1
        i = <optimized out>
        __func__ = "exec_stmt_block"
#12 0x00007f826e52076d in exec_toplevel_block (estate=estate@entry=0x7fff419292c0, block=0x561cded02b20) at pl_exec.c:1633
        rc = <optimized out>
#13 0x00007f826e521022 in plpgsql_exec_function (func=func@entry=0x561cdecfa7c0, fcinfo=fcinfo@entry=0x7fff41929470, simple_eval_estate=simple_eval_estate@entry=0x561cded11090, simple_eval_resowner=simple_eval_resowner@entry=0x561cdec5d5f0, procedure_resowner=procedure_resowner@entry=0x561cdec5d5f0, atomic=<optimized out>) at pl_exec.c:622
        estate = {func = 0x561cdecfa7c0, trigdata = 0x0, evtrigdata = 0x0, retval = 0, retisnull = true, rettype = 0, fn_rettype = 2278, retistuple = false, retisset = false, readonly_func = false, atomic = false, exitlabel = 0x0, cur_error = 0x0, tuple_store = 0x0, tuple_store_desc = 0x0, tuple_store_cxt = 0x0, tuple_store_owner = 0x0, rsi = 0x0, found_varno = 0, ndatums = 4, datums = 0x561cded09ec0, datum_context = 0x561cdecf9d40, paramLI = 0x561cded09e78, simple_eval_estate = 0x561cded11090, simple_eval_resowner = 0x561cdec5d5f0, procedure_resowner = 0x561cdec5d5f0, cast_hash = 0x561cded150b0, stmt_mcontext = 0x561cded4f3f0, stmt_mcontext_parent = 0x561cdecf9d40, eval_tuptable = 0x0, eval_processed = 1, eval_econtext = 0x561cded11298, err_stmt = 0x561cded025c0, err_var = 0x0, err_text = 0x0, plugin_info = 0x0}
        plerrcontext = {previous = 0x7fff41929aa0, callback = 0x7f826e518d90 <plpgsql_exec_error_callback>, arg = 0x7fff419292c0}
        i = <optimized out>
        rc = <optimized out>
        __func__ = "plpgsql_exec_function"
#14 0x00007f826e52ae7c in plpgsql_inline_handler (fcinfo=<optimized out>) at pl_handler.c:368
        _save_exception_stack = <optimized out>
        _save_context_stack = <optimized out>
        _local_sigjmp_buf = {{__jmpbuf = {140734293513696, 5243518032288499632, 94681997218144, 140734293513696, 94681996507624, 94681997217880, -5243820561868961872, -5206042992481843280}, __mask_was_saved = 0, __saved_mask = {__val = {94681997246304, 140734293513520, 140198173514040, 13587, 94681962543572, 94681996768000, 140198173423768, 140198173388096, 140198173423768, 58355720650752, 94681997191488, 13568, 0, 13221425031046758401, 94681963721045, 4}}}}
        _do_rethrow = <optimized out>
        fake_fcinfodata = {fcinfo = {flinfo = 0x7fff41929440, context = 0x0, resultinfo = 0x0, fncollation = 0, isnull = false, nargs = 0, args = 0x7fff41929490}, fcinfo_data = "@\224\222A\377\177", '\000' <repeats 25 times>}
        fake_fcinfo = <optimized out>
        codeblock = 0x561cdecf9160
        func = 0x561cdecfa7c0
        flinfo = {fn_addr = 0x0, fn_oid = 0, fn_nargs = 0, fn_strict = false, fn_retset = false, fn_stats = 0 '\000', fn_extra = 0x0, fn_mcxt = 0x561cdecf9d40, fn_expr = 0x0}
        simple_eval_estate = 0x561cded11090
        simple_eval_resowner = <optimized out>
        retval = <optimized out>
        rc = <optimized out>
        __func__ = "plpgsql_inline_handler"
#15 0x0000561cdcbe88b4 in FunctionCall1Coll (flinfo=0x7fff419295e0, collation=<optimized out>, arg1=<optimized out>) at fmgr.c:1110
        fcinfodata = {fcinfo = {flinfo = 0x7fff419295e0, context = 0x0, resultinfo = 0x0, fncollation = 0, isnull = false, nargs = 1, args = 0x7fff419295b0}, fcinfo_data = "\340\225\222A\377\177", '\000' <repeats 23 times>, "\177\001\000`\221\317\336\034V\000\000\000\234\274\334\034V\000"}
        fcinfo = 0x7fff41929590
        result = <optimized out>
        __func__ = "FunctionCall1Coll"
        __errno_location = <optimized out>
#16 0x0000561cdcbe8fe3 in OidFunctionCall1Coll (functionId=<optimized out>, collation=collation@entry=0, arg1=94681997218144, arg1@entry=281479271691539) at fmgr.c:1388
        flinfo = {fn_addr = 0x7f826e52ad40 <plpgsql_inline_handler>, fn_oid = 13587, fn_nargs = 1, fn_strict = true, fn_retset = false, fn_stats = 1 '\001', fn_extra = 0x0, fn_mcxt = 0x561cdec4b600, fn_expr = 0x0}
#17 0x0000561cdc8c7e75 in ExecuteDoStmt (pstate=0x0, pstate@entry=0x561cdecf9058, stmt=stmt@entry=0x561cdec4b9e8, atomic=atomic@entry=false) at functioncmds.c:2144
        codeblock = 0x1000100003513
        arg = <optimized out>
        as_item = <optimized out>
        language_item = <optimized out>
        language = <optimized out>
        laninline = <optimized out>
        languageTuple = <optimized out>
        languageStruct = <optimized out>
        __func__ = "ExecuteDoStmt"
#18 0x0000561cdcab64c6 in standard_ProcessUtility (pstmt=0x561cdecf7828, queryString=0x561cdecf7420 "do\n$$\n    declare\n        cursor int;\n        id text;\n        row_counter int := 0;\n    begin\n        drop table if exists test;\n        create table test(\n", ' ' <repeats 29 times>, "id text\n      "..., readOnlyTree=<optimized out>, context=PROCESS_UTILITY_TOPLEVEL, params=0x0, queryEnv=0x0, dest=0x561cdcecab20 <donothingDR>, qc=0x7fff419299e0) at utility.c:714
        parsetree = 0x561cdec4b9e8
        isTopLevel = <optimized out>
        isAtomicContext = false
        pstate = 0x561cdecf9058
        readonly_flags = <optimized out>
        __func__ = "standard_ProcessUtility"
#19 0x0000561cdcab4d81 in PortalRunUtility (portal=portal@entry=0x561cdec9f790, pstmt=pstmt@entry=0x561cdecf8038, isTopLevel=<optimized out>, setHoldSnapshot=setHoldSnapshot@entry=false, dest=dest@entry=0x561cdcecab20 <donothingDR>, qc=qc@entry=0x7fff419299e0) at pquery.c:1158
No locals.
#20 0x0000561cdcab4e8f in PortalRunMulti (portal=portal@entry=0x561cdec9f790, isTopLevel=isTopLevel@entry=true, setHoldSnapshot=setHoldSnapshot@entry=false, dest=0x561cdcecab20 <donothingDR>, dest@entry=0x561cdec20360, altdest=0x561cdcecab20 <donothingDR>, altdest@entry=0x561cdec20360, qc=qc@entry=0x7fff419299e0) at pquery.c:1315
        pstmt = 0x561cdecf8038
        stmtlist_item__state = {l = 0x561cdec4c1b0, i = 0}
        active_snapshot_set = false
        stmtlist_item = 0x561cdec4c1c8
#21 0x0000561cdcab53d1 in PortalRun (portal=portal@entry=0x561cdec9f790, count=9223372036854775807, isTopLevel=isTopLevel@entry=true, run_once=<optimized out>, dest=dest@entry=0x561cdec20360, altdest=altdest@entry=0x561cdec20360, qc=0x7fff419299e0) at pquery.c:791
        _save_exception_stack = 0x7fff41929ac0
        _save_context_stack = 0x7fff41929aa0
        _local_sigjmp_buf = {{__jmpbuf = {1, -1946388135562098768, 94681996851088, 94681996328792, 0, 94681996327760, -5243820561984305232, -1946383483128952912}, __mask_was_saved = 0, __saved_mask = {__val = {140734293641813, 140198322149984, 796, 94681996328792, 94681962250274, 1709653270, 310039, 0, 17085207686297854208, 94681996327488, 94681961360192, 94681996327488, 94681996851088, 112, 94681996328792, 0}}}}
        _do_rethrow = <optimized out>
        result = <optimized out>
        nprocessed = <optimized out>
        saveTopTransactionResourceOwner = 0x561cdec5c690
        saveTopTransactionContext = 0x561cdec51e50
        saveActivePortal = 0x0
        saveResourceOwner = 0x561cdec5c690
        savePortalContext = 0x0
        saveMemoryContext = 0x561cdec1fa40
        __func__ = "PortalRun"
#22 0x0000561cdcab2ac6 in exec_execute_message (max_rows=<optimized out>, portal_name=0x561cdec1fb50 "") at postgres.c:2231
        portal = 0x561cdec9f790
        sourceText = 0x561cdec1ff58 "do\n$$\n    declare\n        cursor int;\n        id text;\n        row_counter int := 0;\n    begin\n        drop table if exists test;\n        create table test(\n", ' ' <repeats 29 times>, "id text\n      "...
        prepStmtName = 0x561cdcd4bb3b "<unnamed>"
        was_logged = false
        cmdtaglen = 2
        dest = DestRemoteExecute
        completed = <optimized out>
        qc = {commandTag = CMDTAG_UNKNOWN, nprocessed = 0}
        portalParams = 0x0
        save_log_statement_stats = false
        is_xact_command = false
        msec_str = "\220\313\304\336\034V\000\000\\0\300\334\034V\000\000У\301\336\034V\000\000\000=\000\000\000\000\000"
        params_data = {portalName = 0x561cdeca3178 "", params = 0x0}
        params_errcxt = {previous = 0x0, callback = 0x561cdc9a09d0 <ParamsErrorCallback>, arg = 0x7fff419299f0}
        receiver = 0x561cdec20360
        execute_is_fetch = false
        cmdtagname = <optimized out>
        dest = <optimized out>
        receiver = <optimized out>
        portal = <optimized out>
        completed = <optimized out>
        qc = <optimized out>
        sourceText = <optimized out>
        prepStmtName = <optimized out>
        portalParams = <optimized out>
        save_log_statement_stats = <optimized out>
        is_xact_command = <optimized out>
        execute_is_fetch = <optimized out>
        was_logged = <optimized out>
        msec_str = <optimized out>
        params_data = <optimized out>
        params_errcxt = <optimized out>
        cmdtagname = <optimized out>
        cmdtaglen = <optimized out>
        __func__ = "exec_execute_message"
        __errno_location = <optimized out>
        __errno_location = <optimized out>
        __errno_location = <optimized out>
        __errno_location = <optimized out>
        __errno_location = <optimized out>
#23 PostgresMain (dbname=<optimized out>, username=<optimized out>) at postgres.c:4704
        portal_name = 0x561cdec1fb50 ""
        max_rows = 0
        firstchar = <optimized out>
        input_message = {data = 0x561cdec1fb50 "", len = 5, maxlen = 1024, cursor = 5}
        local_sigjmp_buf = {{__jmpbuf = {0, -5243820562338723920, 94681996563960, 94681961265488, 0, 94681996481376, -5243820562300975184, -1946383485391779920}, __mask_was_saved = 1, __saved_mask = {__val = {4194304, 140734293524224, 732010606731542723, 1039, 16432, 245, 1258425418708, 30, 3, 0, 511101108348, 519691042929, 17085207686297854208, 9, 140198365293696, 15664}}}}
        send_ready_for_query = false
        idle_in_transaction_timeout_enabled = false
        idle_session_timeout_enabled = false
        __func__ = "PostgresMain"
#24 0x0000561cdca2e21a in BackendRun (port=0x561cdec4cb90, port=0x561cdec4cb90) at postmaster.c:4464
No locals.
#25 BackendStartup (port=0x561cdec4cb90) at postmaster.c:4192
        bn = 0x561cdec45360
        pid = <optimized out>
        bn = <optimized out>
        pid = <optimized out>
        __func__ = "BackendStartup"
        __errno_location = <optimized out>
        __errno_location = <optimized out>
        save_errno = <optimized out>
        __errno_location = <optimized out>
        __errno_location = <optimized out>
#26 ServerLoop () at postmaster.c:1782
        port = 0x561cdec4cb90
        i = <optimized out>
        now = <optimized out>
        last_lockfile_recheck_time = 1709653183
        last_touch_time = 1709653183
        events = {{pos = 1, events = 2, fd = 6, user_data = 0x0}, {pos = 45, events = 1024, fd = 0, user_data = 0x770000007c}, {pos = -557726752, events = 22044, fd = 1024, user_data = 0x561cdcf0f220 <errordata>}, {pos = -1886478080, events = 3977959902, fd = -557726752, user_data = 0x0}, {pos = 1100127536, events = 32767, fd = -588189152, user_data = 0x0}, {pos = 1100127640, events = 32767, fd = 327680, user_data = 0xffffffffffffffff}, {pos = 7, events = 0, fd = -1, user_data = 0x7fff41929d98}, {pos = -128, events = 4294967295, fd = 1100127655, user_data = 0x0}, {pos = -557540136, events = 22044, fd = 2041953447, user_data = 0x1}, {pos = 1100127740, events = 32767, fd = 1100127744, user_data = 0x7fff41929da4}, {pos = 0, events = 0, fd = -557540135, user_data = 0x6e}, {pos = -1886478080, events = 3977959902, fd = 0, user_data = 0x7fff41929f64}, {pos = 1100129664, events = 32767, fd = 2041957163, user_data = 0x0}, {pos = 0, events = 16777216, fd = 1100129792, user_data = 0xed1ad9de8f8ea100}, {pos = -588236064, events = 22044, fd = 1100128080, user_data = 0x561cdec49e18}, {pos = 1100129664, events = 32767, fd = 1100128080, user_data = 0x7f8279b1f9b4 <__GI_getaddrinfo+4132>}, {pos = -557542368, events = 22044, fd = 0, user_data = 0x7fff4192a5e8}, {pos = 1, events = 0, fd = 0, user_data = 0x0}, {pos = 0, events = 0, fd = 0, user_data = 0x7fff41929fa0}, {pos = 0, events = 22044, fd = 2008707840, user_data = 0x7fff4192a040}, {pos = -589876736, events = 22044, fd = -557505616, user_data = 0x1}, {pos = -557505616, events = 22044, fd = 1100129520, user_data = 0x561cdcc31b00 <fmtint+768>}, {pos = -557539760, events = 22044, fd = 0, user_data = 0x0}, {pos = 1364412240, events = 875900492, fd = 1996501555, user_data = 0x7fff00000002}, {pos = 1100128080, events = 32767, fd = 0, user_data = 0x0}, {pos = 0, events = 0, fd = 1100129648, user_data = 0x561cdcc31f6f <dopr+1119>}, {pos = 0, events = 0, fd = 0, user_data = 0x7f8277ba7340}, {pos = 2, events = 22044, fd = -589473832, user_data = 0x7fff4192a600}, {pos = 0, events = 22044, fd = 2041284545, user_data = 0x0}, {pos = 0, events = 0, fd = 10, user_data = 0x0}, {pos = 16777216, events = 0, fd = 0, user_data = 0x0}, {pos = 0, events = 0, fd = 0, user_data = 0x0}, {pos = 1, events = 6, fd = 0, user_data = 0x1100000002}, {pos = 0, events = 1, fd = 3, user_data = 0x100000000}, {pos = 0, events = 0, fd = 0, user_data = 0x0}, {pos = 0, events = 0, fd = 0, user_data = 0x0}, {pos = 0, events = 0, fd = 0, user_data = 0x0}, {pos = 0, events = 0, fd = 0, user_data = 0x0}, {pos = 1100128320, events = 32767, fd = 1024, user_data = 0x561cdec473a0}, {pos = 2041129917, events = 32642, fd = 30, user_data = 0xbe9512}, {pos = 1, events = 0, fd = 33152, user_data = 0x8a}, {pos = 0, events = 0, fd = 0, user_data = 0x1000}, {pos = 0, events = 0, fd = 1709651729, user_data = 0x0}, {pos = 0, events = 0, fd = 109038768, user_data = 0x65e73cbf}, {pos = 109038768, events = 118, fd = 32816, user_data = 0x0}, {pos = 0, events = 0, fd = -557550688, user_data = 0xed1ad9de8f8ea100}, {pos = 513, events = 0, fd = 2052, user_data = 0x40}, {pos = 8, events = 0, fd = 1, user_data = 0x780000007c}, {pos = 118, events = 155, fd = -1886478080, user_data = 0x7fff4192a210}, {pos = 2042817664, events = 32642, fd = 32816, user_data = 0x802}, {pos = -591089221, events = 22044, fd = -590063424, user_data = 0x561cdcd451e8}, {pos = 2041287266, events = 32642, fd = 0, user_data = 0x7fff4192a2a0}, {pos = 32768, events = 0, fd = 8, user_data = 0x561cdcc4b1bb}, {pos = 2041553675, events = 32642, fd = 30, user_data = 0xbe90cb}, {pos = 1, events = 0, fd = 16832, user_data = 0x8a}, {pos = 0, events = 0, fd = 0, user_data = 0x1000}, {pos = 0, events = 0, fd = 1709651608, user_data = 0x11c7043fa}, {pos = 1709651608, events = 0, fd = 449118996, user_data = 0xed1ad9de8f8ea100}, {pos = 449118996, events = 0, fd = -128, user_data = 0x0}, {pos = -591089221, events = 22044, fd = -591089702, user_data = 0x7f8279ab862b <_int_malloc+3195>}, {pos = -557530608, events = 22044, fd = 2042817760, user_data = 0x7}, {pos = 15, events = 0, fd = -591089221, user_data = 0x1000}, {pos = 0, events = 0, fd = -1173536573, user_data = 0x10f}, {pos = 4144, events = 0, fd = 64, user_data = 0x7000000101}}
        nevents = <optimized out>
        __func__ = "ServerLoop"
#27 0x0000561cdca2f202 in PostmasterMain (argc=argc@entry=3, argv=argv@entry=0x561cdec1a320) at postmaster.c:1466
        opt = <optimized out>
        status = <optimized out>
        userDoption = <optimized out>
        listen_addr_saved = true
        i = <optimized out>
        output_config_variable = <optimized out>
        __func__ = "PostmasterMain"
#28 0x0000561cdc786281 in main (argc=3, argv=0x561cdec1a320) at main.c:198
        do_check_root = <optimized out>

```

**Comment by okbob** [comment]
should be fixed by 6c197a955f638784cc4e9710eeec8e4effba9def

**Comment by chmyaf** [comment]
Yeah, thanks!
